user.go

```diff
@@ -40,6 +41,11 @@ type userRequest struct {
 	UserOptions *UserOptions `json:"user,omitempty"`
 }
 
+type ListChildUsersResult struct {
+	Children []*User `json:"children,omitempty"`
```

Comment:
This should probably be []*Children. You will need to define a new struct for Children as it has different attributes than User